src/main/java/bio/terra/dao/StudyDao.java

```diff
@@ -0,0 +1,66 @@
+package bio.terra.dao;
+
+import bio.terra.configuration.DataRepoJdbcConfiguration;
+import bio.terra.metadata.Study;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.UUID;
+
+@Repository
+public class StudyDao extends MetaDao<Study> {
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    @Autowired
```

Comment:
There is nothing technically wrong with these @Autowired's.  The Spring advice is to put all of your autowiring into your constructor. IIRC, it makes resolution more efficient and startup faster.

Reply:
done